app/Console/Kernel.php

```diff
@@ -25,11 +25,19 @@ protected function schedule(Schedule $schedule)
     {
         $schedule->command('db:seed:backup --all') // use custom command to make sure that the commands are chained
             ->daily() // Run daily at midnight
+            ->environments(['staging', 'dev', 'local', 'production']) // only run for APP_ENV tagged staging, dev, local, or production
+            ->timezone('America/Los_Angeles') // Run as PST timezone
+            ->onOneServer(); // run only on a single server at once
+
+        $schedule->command('app:refresh-dev') // use custom hcommand to make sure that te commands are chained
+            ->dailyAt('00:15') // Run daily at 12:15 am
             ->environments(['staging', 'dev', 'local']) // only run for APP_ENV tagged staging, dev, or local
+            ->timezone('America/Los_Angeles') // Run as PST timezone
             ->onOneServer(); // run only on a single server at once
```

Comment:
I guess this gets added to all deployments. Is it possible to only add this to the scheduling on the dev deploy?

Reply:
Yeah we can.

Reply:
Limited to **dev** now.